app/models/restaurant.rb

```diff
@@ -0,0 +1,7 @@
+class Restaurant < ApplicationRecord
+  validates :name, presence: true
+  validates :street, presence: true
+  validates :zip_code, presence: true
+  validates :city, presence: true
+  validates :food_style, presence: true
+end
```

Comment:
Not a blocker, I just want you guys to know there's another way to write this:
`validates_presence_of :name`, etc.